migrate.sh

```diff
@@ -9,7 +9,9 @@ do
     echo "Applying migration $file"
     chmod u+x $file
     /bin/bash $file
-    mv $file ./applied_migrations
+    echo ""
+    echo "deleting migration file $file"
+    rm $file
```

Comment:
did a thing, hope its ok 👀

Reply:
I think the latest version of scaffolds does that now,  keeping the migrations is pretty pointless. You can't undo them